doc/api/n-api.md

```diff
@@ -5377,6 +5391,11 @@ This API is used to indicate that the event loop running on the main thread
 should not exit until `func` has been destroyed. Similar to [`uv_ref`][] it is
 also idempotent.
 
+Neither `napi_ref_threadsafe_function` nor `napi_unref_threadsafe_function`
+marks the thread-safe functions to be able to be destroyed nor prevent them
+from been destroyed, try `napi_acquire_threadsafe_function` and
+`napi_release_threadsafe_function` instead.
+
```

Comment:
```suggestion
Neither does `napi_unref_threadsafe_function` mark the thread-safe functions as
able to be destroyed nor does `napi_ref_threadsafe_function` prevent it from
being destroyed. `napi_acquire_threadsafe_function` and
`napi_release_threadsafe_function` are available for that purpose.
```

IIRC our style is to avoid addressing the reader.